torch/export/_trace.py

```diff
@@ -988,16 +972,32 @@ def forward(self, *args, **kwargs):
         fake_params_buffers = make_fake_params_buffers(
             fake_mode, _get_params_buffers(mod)
         )
+
         with fake_mode:
-            ep_non_strict = _export_non_strict(
-                mod,
-                fake_args,
-                fake_kwargs,
-                fake_params_buffers,
-                constant_attrs,
-                pre_dispatch=pre_dispatch,
-                transform=_tuplify_outputs,
-            )
+            with _fakify_script_objects(mod, fake_args, fake_kwargs, fake_mode) as (
```

Comment:
Should probably move `_fakify_script_objects` into `_export_non_strict` since strict mode would also benefit from this. and also move the patching-back logic of ep_non_strict.constants into _export_non_strict

Reply:
Sounds legit, are you doing this in this PR or another one?

Reply:
I'll do it in this one.

Reply:
I changed my mind. We probably should wait for dynamo support landed first.